packages/inspector/package.json

```diff
@@ -8,13 +8,17 @@
   "dependencies": {
     "@hex-engine/core": "^0.3.1",
     "lodash.debounce": "^4.0.8",
+    "lodash.get": "^4.4.2",
+    "lodash.setwith": "^4.3.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-state-tree": "^0.1.2",
```

Comment:
I think you can remove react-state-tree from the deps now, right?

Reply:
Yep, removed in following commit.